.readthedocs.yaml

```diff
@@ -0,0 +1,33 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for 
+# details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
```

Comment:
Why not 3.10?

Reply:
I copied the version in the `setup.cfg` file but there is no reason it couldn't be 3.11